src/app/zap-templates/zcl/data-model/chip/binding-cluster.xml

```diff
@@ -18,6 +18,7 @@ limitations under the License.
   <domain name="CHIP" />
 
   <struct name="TargetStruct" isFabricScoped="true">
+    <cluster code="0x0752"/> <!-- Joint Fabric Datastore -->
```

Comment:
As above.

Reply:
Copied the enum definition to the JF datastore cluster and submitted the update in the most recent change set.